src/sentry/integrations/slack/message_builder/prompt.py

```diff
@@ -4,19 +4,30 @@
 
 from .base.block import BlockSlackMessageBuilder
 
-LINK_IDENTITY_MESSAGE = "Link your Slack identity to Sentry to unfurl Discover charts."
+EPHEMERAL_LINK_IDENTITY_MESSAGE = "Link your Slack identity to Sentry to unfurl Discover charts."
+NON_EPHEMERAL_LINK_IDENTITY_MESSAGE = "You sent an unfurlable link in <#{}>. Please link your Slack identity to Sentry to unfurl Discover charts."
```

Comment:
nit: kind of personal preference comment but I prefer using lambda and "f" for formatting and including a variable name instead of <#{}> . I think it's more clear that way.

Reply:
the <# is so slack formats the channel id into the real channel name. i wanted to keep this a constant outside, so i call .format() on the message to inject the channel_id.

i also forgot to link the video recording, which i did just now.